README.rst

```diff
@@ -39,6 +39,14 @@ Add the tables to the db::
 
 **Warning**: An initial data fixture is included that creates an admin_sso.Assignment to assign any user with an incuna.com email to the Admin user.
 
+If django-rewrite-external-links_ is being used, make sure to allow anonymous access to the external link redirect page by adding the following in ``settings.py``::
```

Comment:
I think I would make it more generic and say: to allow anonymous access links as for example external links you can use `LOGIN_EXEMPT_URLS`.